selfdrive/car/body/values.py

```diff
@@ -22,13 +19,12 @@ def __init__(self, CP):
     pass
 
 
-class CAR(StrEnum):
-  BODY = "COMMA BODY"
-
-
-CAR_INFO: Dict[str, CarInfo] = {
-  CAR.BODY: CarInfo("comma body", package="All"),
-}
+class CAR(Platforms):
+  BODY = (
+    "COMMA BODY",
+    CarInfo("comma body", package="All"),
+    dbc_dict('comma_body', None),
+  )
```

Comment:
ah yes this is what I sort of meant since you need multiple CarInfo's per platform. I would just make it a list or tuple by definition so there's no dynamic type checking on the CarInfo part